docs/command/mongocli-iam-projects-create.txt

```diff
@@ -53,6 +69,10 @@ Options
      - string
      - false
      - Unique 24-hexadecimal digit string that identifies the Atlas user account to be granted the Project Owner role on the specified project.
+   * - --withoutDefaultAlertSettings
+     - 
+     - false
+     - Flag that indicates whether to create the new project with the default alert settings enabled. This flag defaults to false.
```

Comment:
Flag that indicates whether to create a new project with or without alerts settings. The flag defaults to false. In this case, Atlas creates a new project without the default alert settings. This is useful if you create projects programmatically and want to create your own alerts instead of using the default alert settings. To create a project that uses the default alert settings, set this flag to true.